shell.nix

```diff
@@ -12,6 +12,7 @@ in pkgs.mkShell {
 
     # Shells
     pkgs.bash
+    pkgs.cmd
```

Comment:
Nix is a Linux package manager used for building the CI environment, it doesn't have `cmd.exe` in it.

Reply:
My apologies, brainfart.